locales/hi.ftl

```diff
@@ -0,0 +1,20 @@
+title = Firefox Quantum Release कैंपेन
+project-name = Firefox Quantum Release
+main-title = Firefox स्ट्राइक्स बैक!
+main-description = नये Firefox के बारे में लोगो को बतायें। 14 नवम्बर को शाम साढ़े-सात बजे आइए ऑनलाइन और ट्वीट करिए अपनी मनपसंद राय नए Firefox के बारे में। साथ में #newfirefox लगाना बिलकुल मत भूलियेगा। आप Firefox के ट्वीट्स को रीट्वीट भी कर सकते हैं।
+thunderclap-cta = मदद करें Firefox Quantum की। 
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = रीट्वीट करें ऑफिसियल ट्वीट को
+tweet-1 = नए Firefox को उपयोग में लाएं और जाने की कैसे ये पहले से दो गुना ज्यादा तेज़ बन गया है। 💥 अभी Download करे : https://mzl.la/newfirefox
+tweet-2 = नया Firefox __________ तेज़ है ? डाउनलोड करें और जानें अभी: https://mzl.la/newfirefox । शुक्रीया @Firefox!
+tweet-3 = नया @Firefox आ चुका है, अब पहले से और भी ज्यादा सक्षम, बेहतर और तेज़।  कैसे? Download करिये और खुद जान लीजिये :  https://mzl.la/newfirefox  
+own-tweet-cta = अपने शब्दों को ट्वीट करें 
+hashtag = #newfirefox
+github = GitHub
+legal = कानून
+license = अनुज्ञप्ति
+guidelines = दिशा निर्देश 
+mozilla-description =
+  Mozilla एक नॉन-प्रॉफिट कंपनी है जो जुटी है आपके online experience का रिमोट-कंट्रोल आपको देने के लिए एवं वेब के कल को इंसान की बढ़ोतरी हेतु ढालने में। जाने और :   
+  <a href="https://mozilla.org" target="_blank">mozilla.org</a>
+official-tweet-id = 928033132908986369
```

Comment:
as this is an old tweet and was only used as an example until now, I'm in the process of removing it. Could you please set this string to "tbd"? I will update this tomorrow for all language files once the official tweet is out.

Thanks!